ethereum/contracts/SwapCreator.sol

```diff
@@ -160,11 +174,11 @@ contract SwapCreator is ERC2771Context, Secp256k1 {
     }
 
     // Bob can claim if:
-    // - Alice has set the swap to `ready` or it's past t_0 but before t_1
+    // - (Alice has set the swap to `ready` or it's past timeout0) and it's before timeout1
```

Comment:
Did you mean to place the left parenthesis before the word `or`?

Reply:
no, i wasn't sure how to make this non-ambiguous in natural language lol